agenta-web/src/components/Scripts/GlobalScripts.tsx

```diff
@@ -0,0 +1,34 @@
+import {useState, useEffect, useCallback, ComponentType} from "react"
+import Head from "next/head"
+import {isDemo} from "@/lib/helpers/utils"
+import {dynamicComponent} from "@/lib/helpers/dynamic"
+
+const GlobalScripts = () => {
+    const [CloudScripts, setCloudScripts] = useState<ComponentType | null>(null)
+
+    const initializeScripts = useCallback(() => {
+        const Scripts = dynamicComponent("Scripts/assets/CloudScripts")
+        setCloudScripts(() => Scripts)
+    }, [])
+
+    useEffect(() => {
```

Comment:
1, here let's add a ref to keep track of initialization status. otherwise it may and will try loading it multiple times

Reply:
fixed